src/renderer/components/commands-editors.tsx

```diff
@@ -143,24 +140,22 @@ export class EditorDropdown extends React.Component<
       const { editorMosaic } = appState;
       editorMosaic.addNewFile(id);
       editorMosaic.show(id);
-      editorMosaic.customMosaics.push(id);
     } catch (error) {
       appState.showErrorDialog(error.message);
     }
-  }
+  };
 
-  public async removeCustomEditor(event: React.MouseEvent) {
+  public removeFile = (event: React.MouseEvent) => {
     const { id } = event.currentTarget;
     const { editorMosaic } = this.props.appState;
 
-    console.log(`EditorDropdown: Removing custom editor ${id}`);
-    editorMosaic.removeCustomMosaic(id as EditorId);
-  }
+    editorMosaic.hide(id as EditorId);
+  };
 
-  public onItemClick(event: React.MouseEvent) {
+  public onItemClick = (event: React.MouseEvent) => {
```

Comment:
thought: maybe we should go ahead and refactor our old class functions into arrow functions to avoid the `bind(this)` boilerplate everywhere.

Reply:
Agreed. Arrow functions are an overall better way of doing the same thing.

Reply:
Added in af73e65b544f71d1ee398f7e35e3a83c552c0cbb. The 'Remove' menuitems now fully remove the file, not just hide it.